.env.example

```diff
@@ -10,8 +10,10 @@ LIBRARY_ROOT=./video
 # It will automatically be cleaned up on kyoo's startup/shutdown/runtime.
 CACHE_ROOT=/tmp/kyoo_cache
 LIBRARY_LANGUAGES=en
-# A pattern (regex) to ignore video files.
-LIBRARY_IGNORE_PATTERN=".*/[dD]ownloads?/.*"
+# If this is true, kyoo will prefer to download the media in the original language of the item.
+MEDIA_PREFER_ORIGINAL_LANGUAGE=false
+# A pattern (regex) to ignore files.
+LIBRARY_IGNORE_PATTERN=".*/[dD]ownloads?/.*|.*\.(mp3|srt|jpg|jpeg|png|gif|bmp|tiff|svg)$|.*[Tt][Rr][Aa][Ii][Ll][Ee][Rr].*"
```

Comment:
you do not need to ignore all those file extensions, we only check for video files (it uses's guessit's mimetype to filter `video/`)

Reply:
In fact, they are common extensions of extra files that are usually in the same directory as the movie, as I come from jellyfin there are a lot of trickplay files that end up making the scan super slow because guessit is invoked unnecessarily for each thumbnail. I will remove this here, maybe add this as a "tip" in the documentation in another MR.